app/models/service_template.rb

```diff
@@ -400,8 +400,8 @@ def self.create_from_options(options)
   end
   private_class_method :create_from_options
 
-  def provision_request(user, options = nil)
-    provision_workflow(user, options).submit_request
+  def provision_request(user, options = nil, request_options = nil)
```

Comment:
@bzwei 
https://github.com/ManageIQ/manageiq/pull/13972 implemented this method to "get" a provision_request from a service template.